src-tauri/src/constants.rs

```diff
@@ -58,3 +58,7 @@ pub const NORTHSTAR_DLL: &str = "Northstar.dll";
 
 // Profile that Northstar defaults to and ships with
 pub const NORTHSTAR_DEFAULT_PROFILE: &str = "R2Northstar";
+
+// List of valid compatibility tools that Northstar can be launched with
```

Comment:
I switched to doc comments in 7f26fc545503c523a6ce2bf2fc198190ec5720a3 so this should also be updated accordingly ^^

```suggestion
/// List of valid compatibility tools that Northstar can be launched with
```